templates/ts-apollo-fullstack/client/src/components/comment/CreateComment.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { useCreateCommentMutation } from '../../generated-types';
+
+
+type createCommentProps = {
+    noteId:string,
+    addCommentState:any
+  }
+
+
+const CreateComment = ({noteId,addCommentState}:createCommentProps) => {
+    const [createComment] = useCreateCommentMutation();
+    const [newCommentTitle, setNewCommentTitle] = useState("");
+    const [newCommentDescription, setNewCommentDescription] = useState("");
+
+    return(
+        <div>
+            <fieldset>
+                <legend>Create New Comment</legend>
```

Comment:
While this work is there any reason why we do not use react elements? It will be much easier to style that after